### PR TITLE
fix(fiber): Fix fiber returning 500 internal server error

### LINKED ISSUE #162: Fiber Example Returns 500 Internal Server Error
I was having issues after implementing my lambda with the fiber adapter returning 500 from `fiberLambda.ProxyWithContext`. 

So i blew it all away and built the example alone, uploaded it to a brand new function in AWS, and the behavior is exactly the same. 

When I dump routes, I don't even see the expected `GET` that we wire up.

Example: https://github.com/awslabs/aws-lambda-go-api-proxy/blob/bf16974733133c4d9c4cf6778503f60a703ced9b/examples/fiber/main.go

### LINKED ISSUE #175: fiber adapter broken since v0.14.0 release
Ever since[ this release ](https://github.com/awslabs/aws-lambda-go-api-proxy/releases/tag/v0.14.0) any attempt to upgrade results in `Internal Server Error` on lambda and no logs in cloudwatch.  This results in all of my Lambda+Fiber+Go projects going down until I hard-code this back to v0.13.3

### PR DESCRIPTION
**Pull request by marvin-w**
Fixes #175
Fixes #162

*Description of changes:*

https://github.com/awslabs/aws-lambda-go-api-proxy/commit/fde59295b7aa31f7d8f65997da1a94bad8f69303 caused a regression in the fiber adapter logic as we previously would try to get the source IP from nil, which internally would just return an empty string.

Since above commit, the RemoteAddr is not nil anymore, but is lacking the required port that is needed in order to parse it as a *TCPAddr

This commit makes sure the port section is always added if it does not exist. Furthermore, it adds a test that prevents this from happening again and we now have logging in place.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

=> I do.

### CODE REVIEW

File: fiber/adapter.go [hunk 1]
```diff
@@ -103,7 +106,7 @@ func (f *FiberLambda) adaptor(w http.ResponseWriter, r *http.Request) {
 	defer fasthttp.ReleaseRequest(req)
 
 	// Convert net/http -> fasthttp request
-	body, err := ioutil.ReadAll(r.Body)
```
Comment (marvin-w) [inline fiber/adapter.go:106]: `ioutil.ReadAll` is deprecated.

File: fiber/adapter.go [hunk 2]
```diff
@@ -129,8 +132,16 @@ func (f *FiberLambda) adaptor(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	remoteAddr, err := net.ResolveTCPAddr("tcp", r.RemoteAddr)
+	// We need to make sure the net.ResolveTCPAddr call works as it expects a port
+	addrWithPort := r.RemoteAddr
+	if !strings.Contains(r.RemoteAddr, ":") {
+		addrWithPort = r.RemoteAddr + ":80" // assuming a default port
```
Comment (HappyZombies) [inline fiber/adapter.go:138]: Should we be hard coding this port? Regardless, shouldn't we assume port 443 instead for https?

Comment (pinoniq) [reply fiber/adapter.go:138]: Do we know why this net.ResolveTCPAddr is even done?

Comment (marvin-w) [reply fiber/adapter.go:138]: Fiber uses fasthttp and fasthttp requires a ctx and the ctx requires a *[TCPAddr](https://pkg.go.dev/net#TCPAddr).

Comment (pinoniq) [reply fiber/adapter.go:138]: Right. Before the change you mentioned that broke this. It was indeed set to nil. It got set to a value in a change that added ALB support.

Pretty sure we can just this to nil again here, no?

Comment (marvin-w) [reply fiber/adapter.go:138]: No. Then you don't have the source ip in the logs.

**Pull request merged**